Makefile

```diff
@@ -108,6 +108,10 @@ PL011_GENERIC_UART		:= 0
 ENABLE_PMF			:= 0
 # Flag to enable PSCI STATs functionality
 ENABLE_PSCI_STAT	:= 0
+# Whether code and read-only data should be put on separate memory pages.
+# The platform Makefile is free to override this value.
+SEPARATE_CODE_AND_RODATA	:= 0
+
```

Comment:
SEPARATE_CODE_AND_RODATA is a trade-off between security and memory consumption.
Setting this build flag to 1 by default would increase the memory footprint of all BL images by default
and on its own it doesn't bring any additional security if the platform port doesn't make use of
the finer-grained view of the image layout and map the code as executable and the read-only
data as execute-never.